Specify that az sf is still supported for ARM operations

prasadker opened the pull request:
Discussed with Mani and we concluded that it is good for customer to know that we still support "az sf" for ARM operations. They might get confused with the usage of sfctl and az sf.

azurecla commented:
Hi __@rashimg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

prasadker committed:
Specify that az sf is still supported for ARM operations

Discussed with Mani and we concluded that it is good for customer to know that we still support "az sf" for ARM operations. They might get confused with the usage of sfctl and az sf.

mani-ramaswamy commented on articles/service-fabric/service-fabric-application-lifecycle-sfctl.md:
```diff
@@ -172,6 +172,8 @@ When an application upgrade is in progress, the status can be retrieved using th
 Finally, if an upgrade is in progress and needs to be canceled, you can use
 the `sfctl application upgrade-rollback` to roll back the upgrade.
 
+NOTE - We recommend using sfctl for cluster management, howerver, for ARM, SF CLI - https://docs.microsoft.com/en-us/cli/azure/sf/cluster?view=azure-cli-latest (az sf) can still be used. 
```
NOTE - We recommend using sfctl for cluster management, howerver, Azure Resource Manager (ARM) operations are supported in the SF CLI - https://docs.microsoft.com/en-us/cli/azure/sf/cluster?view=azure-cli-latest (az sf).

samedder replied:
Please fix this to be a proper markdown link. It's also confusing since the SF CLI is sfctl. Please use the Azure CLI when referring to `az sf` commands.
